ivy_tests/test_ivy/test_frontends/test_paddle/test_fft.py

```diff
@@ -180,6 +181,40 @@ def test_paddle_ifftshift(
     )
 
 
+@handle_frontend_test(
+    fn_tree="paddle.fft.ihfft2",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=2,
+        min_dim_size=2,
+    ),
+)
+def test_paddle_ihfft2(
+    dtype_x_axis,
+    frontend,
+    backend_fw,
+    test_flags,
+    fn_tree,
+):
+    input_dtypes, x, axis = dtype_x_axis
+    shape = ivy.array(x).shape
+    x = ivy.reshape(x, shape)  # reshape into a 2 dimensional array.
+
+    for norm in ["backward", "forward", "ortho"]:
+        ivy.astype(
```

Comment:
This is redundant as in the function test body we only need to call `helpers.test_frontend_function(*args, **kwargs)`

Reply:
Hello Hirwa, thank you for leaving a review. I will leave out the for loop line then, "for norm in ["backward", "forward", "ortho"]:".  So, is the ihfft2 function in the fft.py alright?